src/topology/uniform_space/basic.lean

```diff
@@ -1217,6 +1217,59 @@ lemma to_topological_space_inf {u v : uniform_space α} :
   (u ⊓ v).to_topological_space = u.to_topological_space ⊓ v.to_topological_space :=
 by rw [to_topological_space_Inf, infi_pair]
 
+section uniform_continuous_infi
+
+local notation `unif_cont` := @uniform_continuous _ _
```

Comment:
You could also use `@@uniform_continuous` directly below.

Reply:
Oooooooh I didn't know about that. Do we have a doc page about it ?